src/olcs/contrib/manager.js

```diff
@@ -128,6 +133,7 @@ olcs.contrib.Manager = class {
     const scene = this.ol3d.getCesiumScene();
     this.configureForUsability(scene);
     this.configureForPerformance(scene);
+    this.dispatchEvent(ol.events.EventType.LOAD)
```

Comment:
Please directly use strings. Usage of `ol.events.EventType` will soon go away.
In addition, I have the feeling the event will be dispatched too early (the manager will not yet be ready). It might be fine to put the dispatch in a `setTimeout` call.

Reply:
Ok so my last commit is useless too ? I just put 'toggle' as parameter ?

Reply:
> It might be fine to put the dispatch in a setTimeout call.

In my example it looks it's loaded on time, ol3d object is ready in my event callback, but i will secure it with a timeout if you think it's better.